tests/prune.test.mjs

```diff
@@ -46,6 +47,27 @@ describe('ARNS Record Pruning', () => {
     sharedMemory = transferMemory;
   });
 
+  it('messages with more than 100 bytes of data will throw an error', async () => {
```

Comment:
I think worthwhile to have a test that we can eval large messages

Reply:
[4d04db8](https://github.com/ar-io/ar-io-network-process/pull/381/commits/4d04db8bf5fa3d0d46a59411f65d70baff1fc895)